src/node.cc

```diff
@@ -3746,62 +3746,39 @@ Environment* CreateEnvironment(Isolate* isolate,
 }
 
 
-int Start(int argc, char** argv) {
-  PlatformInit();
-
-  const char* replaceInvalid = secure_getenv("NODE_INVALID_UTF8");
-
-  if (replaceInvalid == nullptr)
-    WRITE_UTF8_FLAGS |= String::REPLACE_INVALID_UTF8;
-
-  CHECK_GT(argc, 0);
-
-  // Hack around with the argv pointer. Used for process.title = "blah".
-  argv = uv_setup_args(argc, argv);
-
-  // This needs to run *before* V8::Initialize().  The const_cast is not
-  // optional, in case you're wondering.
-  int exec_argc;
-  const char** exec_argv;
-  Init(&argc, const_cast<const char**>(argv), &exec_argc, &exec_argv);
-
-#if HAVE_OPENSSL
-  // V8 on Windows doesn't have a good source of entropy. Seed it from
-  // OpenSSL's pool.
-  V8::SetEntropySource(crypto::EntropySource);
-#endif
-
-  V8::InitializePlatform(new Platform(4));
-
-  int code;
-  V8::Initialize();
-
-  // Fetch a reference to the main isolate, so we have a reference to it
+// Entry point for new node instances, also called directly for the main
+// node instance.
+void StartNodeInstance(void *arg) {
```

Comment:
If StartNodeInstance() is not used elsewhere, it should be `static` or `inline` (they amount to the same thing, non-external linkage.)